lib/runtime/wasmer/imports.go

```diff
@@ -371,6 +354,16 @@ func ext_crypto_secp256k1_ecdsa_recover_version_1(context unsafe.Pointer, sig, m
 	message := memory[msg : msg+32]
 	signature := memory[sig : sig+65]
 
+	if signature[64] == 27 {
+		signature[64] = 0
+	}
+
+	if signature[64] == 28 {
+		signature[64] = 1
+	}
+
```

Comment:
Why are we doing this?

Reply:
the last byte of the signature can be either `0/1` or `27/28` depending on what signed it, see spec: `sig: a 32-bit pointer to the buffer containing the 65-byte signature in RSV format. V should be either 0/1 or 27/28.`